setup.py

```diff
@@ -91,6 +91,7 @@
                 'scigraph-deploy=pyontutils.scigraph_deploy:main',
                 'scig=pyontutils.scig:main',
                 'ttlfmt=pyontutils.ttlfmt:main',
+                'interlex=ilxutils.cli:main'
```

Comment:
This needs to be in the ilxutils setup.py, and then please remove the changes from this file.